### DISCUSSION TITLE
[Feature] Skip corrupt/slow to load images

### DISCUSSION THREAD

**Discussion by iannellomarco** [immich-app/immich#12592] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

Implement a toggle feature in the app that allows users to automatically skip media files (images or videos) that are either corrupted or take longer than a specified amount of time (X seconds or minutes) to load. Additionally, add a manual "Skip" button for users to manually bypass problematic media files if needed. This combination will ensure both automatic and user-controlled management of backup interruptions caused by problematic files.

Use Case: During the backup process, some media files may be corrupted or take too long to load, causing delays or stalling the backup. This feature would allow users to automatically skip these files based on predefined time limits. Additionally, a manual "Skip" button will give users direct control to bypass files that are slowing down the process, even if they do not reach the automatic threshold.

### Platform

- [ ] Server
- [X] Web
- [X] Mobile

**Comment by bo0tzz** [comment]
#6173
